lib/web_ui/test/spy.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:ui/src/engine.dart' hide window;
+import 'package:ui/ui.dart';
+
+class PlatformMessage {
```

Comment:
The classes here sound reusable. Let's document them?

Reply:
Done.